os.path/os/path.py

```diff
@@ -79,6 +79,13 @@ def islink(path):
     except OSError:
         return False
 
+def realpath(path):
+    import ffilib
+    libc = ffilib.libc()
+    if isinstance(path, str) or isinstance(path, bytes):
+        realpath_ = libc.func("s", "realpath", "ss")
+        return realpath_(path, 0)
```

Comment:
"0" isn't really valid value for parameter of type "s" (pointer to string). If you want to pass NULL, pass None.

A bigger problem is that this will lead to memory leak, from `man 3 realpath`:

> If resolved_path is specified as NULL, then realpath()  uses  malloc(3)
       to allocate a buffer of up to PATH_MAX bytes to hold the resolved path‐
       name, and returns a pointer to this buffer.  The caller should  deallo‐
       cate this buffer using free(3).

Currently, there's no good pattern to resolve that in pycopy-lib, so I don't ask you to do that. Jut add comment a-la: `# XXX: memory leak, should free() returned pointer, see man realpath.`